cmd/frontend/graphqlbackend/codemod.go

```diff
@@ -267,6 +267,7 @@ func callCodemodInRepo(ctx context.Context, repoRevs *search.RepositoryRevisions
 	// TODO(RVT): Use a separate HTTP client here dedicated to codemod,
 	// not doing so means codemod and searcher share the same HTTP limits
 	// etc. which is fine for now but not if codemod goes in front of users.
+	// Once separated please fix cmd/frontend/graphqlbackend/textsearch:50
```

Comment:
```suggestion
	// Once separated please fix cmd/frontend/graphqlbackend/textsearch:50 in #6586 
```

^ Risky to refer only to the line number. Referencing the PR helps.

Reply:
I filed #6593 to track this TODO